src/tools/rustfmt/tests/target/issue_4110.rs

```diff
@@ -20,6 +20,7 @@ fn bindings() {
                 category,
                 span,
                 &format!("`{}`", name),
+                "function",
```

Comment:
Was this change necessary? I'm curious why a rustfmt test needed to change.

Reply:
I added a new argument in rustc. The rustfmt change is just to keep compatibility.

Reply:
@ytmimi - Technically I don't think it's necessary because we never need to compile these files, but I don't think the extra arg impacts the integrity of the rustfmt test in any way (I assumed it was simply changed based on a find+replace using the ident)

Reply:
Yeah, I blanket searched for all the function calls.